### PR TITLE
DOCS-4525 Release notes for rtf-agent 1.2.60

### PR DESCRIPTION
**Pull request by amagnus**

### CODE REVIEW

File: modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc [hunk 1]
```diff
@@ -6,6 +6,15 @@ endif::[]
 
 In addition to these release notes, see the complete xref:1.0@runtime-fabric::index.adoc[Anypoint Runtime Fabric] documentation.
 
+== 1.2.x - March 26, 2019
+
+This release includes the following fixes and enhancements:
+
+* Added: Support for secure properties.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc]: Support for secure properties for applications?

File: modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc [hunk 2]
```diff
@@ -6,6 +6,15 @@ endif::[]
...
+* Fixed: Security patches on Mule clustering service.
+* Fixed: Enhanced HTTP proxy support with Mule apps.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc]: `Added: Automatic configuration of HTTP proxy for API Gateway applications`

File: modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc [hunk 3]
```diff
@@ -6,6 +6,15 @@ endif::[]
 
 In addition to these release notes, see the complete xref:1.0@runtime-fabric::index.adoc[Anypoint Runtime Fabric] documentation.
 
+== 1.2.x - March 26, 2019
+
+This release includes the following fixes and enhancements:
+
+* Added: Support for secure properties.
+* Fixed: Security patches on Mule clustering service.
```
Comment (jeff-1amstudios) [inline modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc]: `Added: Cluster Service upgraded to Java 8u191`

**Comment by sundippatel** [comment]
This w/ the suggested changes look good to me. Thanks!

File: modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc [hunk 4]
```diff
@@ -6,6 +6,15 @@ endif::[]
 
 In addition to these release notes, see the complete xref:1.0@runtime-fabric::index.adoc[Anypoint Runtime Fabric] documentation.
 
+== 1.2.x - March 26, 2019
+
+This release includes the following fixes and enhancements:
+
+* Added: Support for consuming secure properties in applications.
+* Added: Cluster Service upgraded to Java 8u191.
+* Added: Automatic HTTP proxy configuration for API Gateway applications
```
Comment (sundippatel) [inline modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc]: Simplify to .."API gateways"

Comment (amagnus) [reply modules/ROOT/pages/runtime-fabric/runtime-fabric-release-notes.adoc]: 👍

**Pull request merged**